deliver/lib/deliver/options.rb

```diff
@@ -189,16 +189,18 @@ def self.available_options
                                      optional: true,
                                      short_option: "-l",
                                      verify_block: proc do |value|
+                                       supported_picture_endings = [".png", ".jpg", ".jpeg"]
```

Comment:
i thought it is better twice declard than having it globally in a class? - any other idea, or sample, where stuff like this should go?